concepts/05 Widgets/DateBox/10 Handle the Value Change Event.md

```diff
@@ -35,6 +35,63 @@ If instead of picking a value, an end user types it into the **DateBox**, this v
         // ...
     })
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxDateBox
+            :value="date"
+            type="date"
+            value-change-event="keyup"
+        />
+    </template>
+
+    <script>
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import DxDateBox from 'devextreme-vue/date-box';
+
+    export default {
+        components: {
+            DxDateBox
+        },
+        data() {
+            return {
+                date: new Date()
+            };
+        }
+    }
+    </script>
+
+##### React
+
+    <!-- tab: App.js -->
+    import React from 'react';
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import DateBox from 'devextreme-react/date-box';
+
+    class App extends React.Component {
+        constructor(props) {
+            super(props);
+
+            this.date = new Date();
+        }
+
+        render() {
+            return (
+                <DateBox
+                    defaultValue={this.date}
+                    type="date"
+                    valueChangeEvent="keyup"
+                />
+            );
+        }
+    }
+    export default App;
```

Comment:
What about the `onValueChanged` example from below? Let's add it and also show two-way binding to the `value` option in it, like we did [for the Calendar](https://github.com/DevExpress/devextreme-documentation/pull/353/files#diff-2e1e9256369f8552a879e3d5bdd78278).

Reply:
fixed